py3status/modules/nvidia_temp.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+"""
+Module for displaying NVIDIA GPU temperature
+
+Requires:
+    - nvidia-smi
+
+@author jmdana <https://github.com/jmdana>
+@license GPLv3 <http://www.gnu.org/licenses/gpl-3.0.txt>
```

Comment:
Would you agree to license using BSD instead ? I try to lower the licenses mess in py3status as much as possible.